frontend/lib/package.json

```diff
@@ -66,6 +66,7 @@
     "lodash": "^4.17.21",
     "mapbox-gl": "^1.13.2",
     "marked": "^4.2.12",
+    "mdast-util-find-and-replace": "^3.0.1",
```

Comment:
This dependency is already a transitive dependency used within `remark`. Since we are directly using it within our code, we are adding it to the dependencies here.